core/src/com/unciv/logic/GameSaver.kt

```diff
@@ -20,30 +21,40 @@ object GameSaver {
     const val settingsFileName = "GameSettings.json"
     var saveZipped = false
 
-    @Volatile
-    var customSaveLocationHelper: CustomSaveLocationHelper? = null
+    /**
+     * This is necessary because the Android turn check background worker does not hold any reference to the actual [com.badlogic.gdx.Application],
+     * which is normally responsible for keeping the [Gdx] static variables from being garbage collected.
+     */
+    private lateinit var files: Files
```

Comment:
Would that be a little clearer if we shortened the import and use "gdx.Files" here and below? Reminds me of the name-clashing Gdx Array :see_no_evil: - OK, this one is not _that_ bad.